talent-class/truongnguyen/exercise-in-slide-3.js

```diff
@@ -0,0 +1,14 @@
+/* 1 */
+function checkOddEven(number) {
+  if (number % 2 == 0) {
+    console.log(number * 2);
+  } else console.log(number * 3);
+}
+/* 2 */
+function rankingStudent(math, phy, che) {
+  var avg = (math + phy + che) / 3;
+  if (avg >= 0 && avg < 4) console.log("Yếu");
```

Comment:
Nên có ngoặc, dù chỉ có 1 mệnh đề.
Lí do:
- Ngắn gọn: dễ gây ra lỗi
- Chi tiết: khi thêm log không để ý sẽ dễ làm sai logic của app
- Tham khảo:
+ https://dev.to/programmerbyday/why-you-should-always-use-curly-braces-for-single-statement-blocks-in-c-41pg
+  https://stackoverflow.com/a/12193202/7228412

Reply:
ok a